WordPress/gutenberg
# Keyboard help modal: Fix scrolling issue in some browsers

## What?


Fixes #41500

## Why?


You are unable to scroll the modal in some browsers.

## How?


Adds tabindex to the modal content and moves focus hooks to place focus on content div to avoid bugs with extra tab stop.

## Testing Instructions





1. Find a browser this is not working in.
2. Open the keyboard help modal from Options menu.
3. Verify scrolling works.

## Screenshots or screencast
